lookout/style/typos/tests/test_preparation.py

```diff
@@ -6,24 +6,26 @@
 from gensim.models import FastText
 import pandas
 
+from lookout.style.format.tests import long_test
 from lookout.style.typos.preparation import (get_datasets, prepare_data, train_and_evaluate,
                                              train_fasttext)
 from lookout.style.typos.utils import Columns, read_frequencies, read_vocabulary
 
 
 TEST_DATA_DIR = pathlib.Path(__file__).parent
-FASTTEXT_DUMP_FILE = str(TEST_DATA_DIR / "id_vecs_10.bin")
-VOCABULARY_FILE = str(TEST_DATA_DIR / "test_frequencies.csv.xz")
 
 
 class PreparationTest(unittest.TestCase):
+    @long_test
```

Comment:
Actually, we can remove this because you significantly reduced the test size below